virtualcontext.js

```diff
@@ -32,9 +31,8 @@ function VirtualContext(id, endpoint=null, parent=null)
 	else
 	{
 		this.id = id || null;
-		this.endpoint = endpoint || null;
 		this.parent = parent || null;
-		this.properties = {"readonly": true};
+		this.properties = {"readonly": true, "virtualSrc": src};
 	}
 
 	this.type = Types.CONTEXT;
```

Comment:
Why not a new type?

Reply:
We do not intend to expand the conceptual model, that is, to actually insert a new type. The idea is to be just a context with particular properties. In addition, this implementation avoids specific treatments on other assets, such as, in KES to change the visualization (rectangular instead of circular pattern), specific treatments for parsing and retrieval in rdf2hk.

Reply:
Shouldn't this be handled through inheritance instead?

Reply:
It should, however for some reason, if this assignment is not done, hkbase services do not return the entity type.

Reply:
Odd. It looks like other assets check instances using `instance.type == Types.<desired type's constant string>` instead of `instance instanceOf <desired type>`.